src/index.ts

```diff
@@ -14,8 +14,19 @@ const getOptionalInput = (name: string) => core.getInput(name) || undefined;
     return;
   }
 
+  let githubUserName = getOptionalInput('githubUserName') || `"github-actions[bot]"`;
+  let githubUserEmail = getOptionalInput('githubUserEmail') || `"github-actions[bot]@users.noreply.github.com"`;
```

Comment:
those are not related to github, so I think this would be more appropriate:
```suggestion
  let gitUserName = getOptionalInput('gitUserName') || `"github-actions[bot]"`;
  let gitUserEmail = getOptionalInput('gitUserEmail') || `"github-actions[bot]@users.noreply.github.com"`;
```

Reply:
Makes sense, will update these along with the README.